### DISCUSSION TITLE
Feature Request: Multiple automatic payments

### DISCUSSION THREAD

**Discussion by SquidwardHill** [americanexpress/one-app#812] [Ideas]
Hey there, 

Thanks for all your hard work on the app!

It would be beneficial to set up multiple automatic payments (that could come from the same or different attached bank accounts).

Context: I share this account with a family member and we are paying from individual and shared accounts. Some of these will be recurring, and so it would be nice to set it and forget it for each account. 

This could be related to #810 , because more context about the payment would be helpful.

Cheers!

**Comment by 10xLaCroixDrinker** [comment]
See: https://github.com/americanexpress/one-app/discussions/810#discussioncomment-3900058
